docs/doxygen/pages.dox

```diff
@@ -12,6 +12,10 @@ The library is written in C and designed to be compliant with ISO C90 and MISRA
 It has proofs showing safe memory use and no heap allocation, making it suitable for IoT microcontrollers, but also fully portable to other platforms.
 </p>
 
+> **NOTE**:
+> If your application uses AWS IoT Jobs library and [OTA agent](https://github.com/aws/ota-for-aws-iot-embedded-sdk) to communicate with AWS IoT through a shared MQTT connection, we suggest that you keep the application logic that uses these libraries within a single task/thread.
```

Comment:
The link is pointing to OTA library. Did you mean OTA library?

Reply:
The OTA library contains the `OtaAgent` phrase as can be seen here: https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/main/source/include/ota.h
However, as the library does not officially call itself as OTA agent, I can rename it as OTA library